src/components/Header/Header.tsx

```diff
@@ -1,28 +1,57 @@
-import { NavLink, useLocation } from 'react-router-dom';
-import styles from './Header.module.scss';
+import './Header.scss';
+import { NavLink } from 'react-router-dom';
 import logo from '../../assets/img/logo.png';
 import { Path } from '../../pages/constants/path';
-import Navigation from '../Navigation/Navigation';
+import CommonButton from '../../UI/CommonButton/CommonButton';
+import { useAppDispatch, useAppSelector } from '../../redux/store';
+import MainApiService from '../../api/MainApiService';
+import { updateUser } from '../../redux/reducers/userSlice';
 
 export default function Header() {
-  const { pathname } = useLocation();
+  const { token, refreshToken } = useAppSelector((state) => state.user.user);
+  const dispatch = useAppDispatch();
 
+  const logOut = async () => {
+    const json = await MainApiService.userLogout({
+      refreshToken: refreshToken?.value,
+    });
+    if (json.statusCode !== 0) {
+      dispatch(
+        updateUser({ token: null, refreshToken: null, userPageDto: null }),
+      );
+    }
+  };
   return (
-    <header className={styles.header}>
-      <div className={styles.container}>
-        <div className={styles.inner}>
-          <div className={styles.logo}>
-            {pathname === Path.HOME ? (
-              <img src={logo} alt="Logo" />
-            ) : (
-              <NavLink to={Path.HOME}>
-                <img src={logo} alt="Logo" />
+    <header className="header">
+      <div className="container">
+        <NavLink to={Path.HOME}>
+          <img src={logo} alt="Logo" />
+        </NavLink>
+        <div className="links">
+          <NavLink to="/">О нас</NavLink>
+          <NavLink to="/">Меню тестов</NavLink>
+          <NavLink to="/">Резюме</NavLink>
+        </div>
+        <div className="controls">
+          {token === null ? (
```

Comment:
Maybe better just check token is existed like that: `token ? ()`

Reply:
Fix this